plugin-gradle/plugin/src/test/kotlin/org/jsonschema2dataclass/js2p/GradleVersions.kt

```diff
@@ -52,6 +52,14 @@ class TestGradleVersionHolder {
     companion object {
         @JvmStatic
         fun gradleReleasesForTests(): List<Arguments> = compatibleVersions.map { Arguments.of(it) }
+
+        @JvmStatic
+        fun configurationCacheCompatibleGradleReleasesForTests(): List<Arguments> = compatibleVersions
+            .filter {
+                ComparableGradleVersion(it) >= 6 to 6
```

Comment:
[Configuration cache was introduced in Gradle 6.6](https://blog.gradle.org/introducing-configuration-caching)